### PR TITLE
Issue 4 explain mod repos

### PR DESCRIPTION
**Pull request by asteflova**
In yesterday's review of ssitani's and jherrmann's AsciiDoc tutorial, we agreed it would be good to explain how a modular guide works and provide an example. Coincidentally, this is the subject of https://github.com/redhat-documentation/modular-docs/issues/4 so it would be nice to fix it so that the tutorial can link to the Mod Docs Guide.

This pull request adds a zip archive with a very simplified modular guide (stripped off of any Red Hat specific content, such as master-docinfo.xml because it's intended for upstream) and also a new section to explain how the guide works.

For reference, this is the issue on the adoc tutorial's side: https://gitlab.cee.redhat.com/ccs-internal-documentation/AsciiDocTutorial/issues/4

### CODE REVIEW

File: modular-docs-manual/content/topics/module_a-simple-example-of-a-modular-guide.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[id='a-simple-example-of-a-modular-guide']
+= A Simple Example of a Modular Guide
+
+Download and unzip link:https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/files/simple-mod-guide-repo.zip[an example modular guide]. The `simple-mod-guide-repo` directory included in the ZIP archive contains a very simple example of a modular guide with assemblies and modules. This section explains the contents of the directory.
+
+NOTE: To build the example guide, change into the `simple-mod-guide-repo` directory, issue the *asciidoctor master.adoc* command, and open the resulting `master.html` file in your browser.
+
```
Comment (bhardesty) [inline modular-docs-manual/content/topics/module_a-simple-example-of-a-modular-guide.adoc]: To explain the directory structure, you might want to show the docs/ tree in a code block and use comments and/or callouts to describe what each dir/file does.

Comment (asteflova) [reply modular-docs-manual/content/topics/module_a-simple-example-of-a-modular-guide.adoc]: Thanks, Ben, I like this idea! This is what I did: https://github.com/redhat-documentation/modular-docs/pull/43/commits/1ceefe7cf57b56605a761f32ef94139bfc69fd6b  I kept the comments in the code block short and followed up with a few more detailed notes.

Let me know if you think of a better solution.

Comment (bhardesty) [reply modular-docs-manual/content/topics/module_a-simple-example-of-a-modular-guide.adoc]: Ack - I took a quick peek and it looks good to me!

**Comment by ritz303** [comment]
@asteflova : Looks good, thanks for adding this content!  ACK.  I'm going to leave this PR open for a few more days to give others an opportunity to comment on your update.

**Pull request merged**